apps/stable_diffusion/web/ui/txt2img_ui.py

```diff
@@ -290,9 +293,9 @@ def txt2img_api(
                             )
                             hf_model_id = gr.Textbox(
                                 elem_id="hf_model_id",
-                                placeholder="Select 'None' in the Models dropdown on the left and enter model ID here e.g: SG161222/Realistic_Vision_V1.3",
+                                placeholder="Select 'None' in the Models dropdown on the left and enter model ID here e.g: SG161222/Realistic_Vision_V1.3, https://civitai.com/api/download/models/15236",
```

Comment:
can this just be 15236 ? so they don't have to add https://civitai.com/api/download/models

Reply:
we can call it model number